CMakeLists.txt

```diff
@@ -1,36 +1,42 @@
-cmake_minimum_required(VERSION 2.8)
-project(Kalman)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
I'd prefer retaining backwards-compatibility to 2.8 as it is still the default on (for instance) Ubuntu 14.04 LTS